src/libponyc/ast/ast.c

```diff
@@ -541,7 +541,24 @@ void ast_resetpass(ast_t* ast)
 const char* ast_get_print(ast_t* ast)
 {
   assert(ast != NULL);
-  return token_print(ast->t);
+
+  switch(token_get_id(ast->t))
+  {
+    case TK_STRING:
+    {
+      char* escaped = token_print_escaped(ast->t);
+      size_t len = strlen(escaped);
+      char* s = (char*)ponyint_pool_alloc_size(len + 3);
```

Comment:
its been a bit since i worked with this stuff, but i believe this needs to be freed somewhere.